src/assets/CurrencyRateController.ts

```diff
@@ -65,7 +69,7 @@ export class CurrencyRateController extends BaseController<CurrencyRateConfig, C
 	constructor(config?: Partial<CurrencyRateConfig>, state?: Partial<CurrencyRateState>) {
 		super(config, state);
 		this.defaultConfig = {
-			currentCurrency: 'usd',
+			currentCurrency: this.getCurrentCurrencyFromState(state),
```

Comment:
So why exactly does this work? Where is `this.defaultState` used?

Reply:
Is it that `this.initialize()` calls `this.update(this.initialState);` in the base controller https://github.com/MetaMask/gaba/blob/f9cbda2780da2f7fec651bd2ca25797c3866b0bd/src/BaseController.ts#L93

which then sets the internal state: https://github.com/MetaMask/gaba/blob/f9cbda2780da2f7fec651bd2ca25797c3866b0bd/src/BaseController.ts#L197

Yeah, I think that is it.